src/utils/CvoPointCloud.cpp

```diff
@@ -438,6 +438,9 @@ namespace cvo{
     //positions_.resize(output_uv.size());
     Mat33f intrinsic = calib.intrinsic();
 
+    num_classes_ = raw_image.num_classes();
+    num_geometric_types_ = 2;
+    feature_dimensions_ = raw_image.channels() + 2;
```

Comment:
Just to make sure, if the new CvoPointCloud a template class with template argument num_class and feature_dim? If so we don't need to assign it to be  raw_image.channels() + 2;